src/Calculator.tsx

```diff
@@ -32,30 +32,30 @@ class Calculator extends React.Component {
      * 
      * @param {} recallWeight 
      */
-    getFScore = (recallWeight) => {
+    getFScore = (recallWeight: number) => {
         const squaredRecall = recallWeight^2
         const numerator = (1 + squaredRecall) * this.state.tP
         const denominator = numerator + (this.state.fN * squaredRecall) + this.state.fP
         const f = numerator / denominator
         return f
     }
 
-    onTPChange = e => {
+    onTPChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
> src/Calculator.tsx:40:18 - error TS7006: Parameter 'e' implicitly has an 'any' type.

Same as before, although the type involved is pretty complex. You can figure out what type this is supposed to be by hovering over `onChange` in line 78 where you use this. Feel free to just use `e: any` for complicated types, if you don't want to go through that.